index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="navbar">
+        <div class="logo">
+            <a href="#"><img src="images/Logo Placeholder.svg"></a>
+        </div>
+        <div class="hamburger-menu">
+            <a href="#"><img src="images/Union.svg" alt="hamburger-menu"></a>
+        </div>
+    </div>
+    <div class="header">
+        <div class="primary-text">
+        <h1>
+            Hey There.
+        </h1>
+        <h1>
+            I’m Vijith.
+        </h1>
+    </div>
+
+    <div class="supporting-text">
+        <h2>
+            I'm a Software Developer
+        </h2>
+    </div>
+    
+    <div class="flex-container-img">
+        <div class="hero-img">
+            <img src="images/illustration-header-mobile.svg">
+        </div>
+        <div class="description">
+            <h3>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+            </h3>
```

Comment:
As required , the description should stick to the Figma design 👍 , as represented on the table of picture below 👎 . Also for semantic reason instead of using an `h3` , Kindly use a `p` tag . Let's fix that to get closer to perfection 😄 

| ### Your design | ### Figma design |
| :---: | :---: |
| ![Capture 10](https://user-images.githubusercontent.com/80458199/236236655-b301938d-e4c8-4b4b-9a67-ba3d1c55c9fc.PNG) | ![Capture 11](https://user-images.githubusercontent.com/80458199/236236751-e7b2a1e1-7667-403f-a90e-c2fb3bbb04ad.PNG) | 

The CSS looks like this 👎

```bash 
font-family: 'Poppins';
font-style: normal;
font-weight: 400;
font-size: 16px;
line-height: 24px;
```